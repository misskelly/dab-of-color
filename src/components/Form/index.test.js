import React from 'react';
import { shallow } from 'enzyme';
import UniForm from '.';

describe('UniForm', () => {
  let wrapper;

  it('should match the snapshot', () => {
    wrapper = shallow(<UniForm />);

    expect(wrapper).toMatchSnapshot();
  });
});
