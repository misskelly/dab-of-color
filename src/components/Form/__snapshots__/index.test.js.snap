// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UniForm should match the snapshot 1`] = `ShallowWrapper {}`;
